apps/api/src/app/workflows-v2/util/template-parser/liquid-parser.ts

```diff
@@ -184,7 +190,8 @@ function extractProps(template: any): { valid: boolean; props: string[]; error?:
     return {
       valid: false,
       props: [],
-      error: `Variables must include a namespace (e.g. payload.${validProps[0]})`,
+      // eslint-disable-next-line max-len
+      error: `Invalid variable name missing namespace. Variables must follow the ${DOT_ANNOTATION_MARKDOWN_LINK} (e.g. payload.${validProps[0]})`,
```

Comment:
Is this long text renders nicely in the ui?

Reply:
![image](https://github.com/user-attachments/assets/56429b87-cb94-47e5-82e7-0b0807bb9990)